### PR TITLE
ci: Fix pipeline

### LINKED ISSUE #186: Fix new warnings in Psalm 5
**Describe the bug**
After updating to Psalm 5 there have been multiple warnings of new errors. We need to remove the newly generated baseline and fix it.

**To Reproduce**
1. remove baseline
2. run psalm

**Expected behavior**
No errors when running psalm

### PR DESCRIPTION
**Pull request by transistive**
Also fixes #186

### CODE REVIEW

File: tests/Unit/BoltFactoryTest.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 namespace Laudis\Neo4j\Tests\Unit;
 
 use Bolt\connection\IConnection;
-use Bolt\enum\ServerState;
+use Bolt\protocol\ServerState;
```
Comment (stefanak-michal) [inline tests/Unit/BoltFactoryTest.php:17]: ServerState doesn't exist as class anymore. Protocol instance itself has property serverState.

File: tests/Unit/ParameterHelperTest.php [hunk 2]
```diff
@@ -165,7 +165,7 @@ public function testDateTime(): void
         $date = ParameterHelper::asParameter(new DateTime('now', new DateTimeZone('Europe/Brussels')), ConnectionProtocol::BOLT_V44());
 
         self::assertInstanceOf(DateTimeZoneId::class, $date);
-        self::assertEquals('Europe/Brussels', $date->tz_id);
+        self::assertEquals('Europe/Brussels', $date->tz_id());
```
Comment (stefanak-michal) [inline tests/Unit/ParameterHelperTest.php:168]: This is not correct. `tz_id` is a property, not method.

File: tests/Unit/BoltFactoryTest.php [hunk 3]
```diff
@@ -43,8 +43,9 @@ protected function setUp(): void
         $protocolFactory = $this->createMock(ProtocolFactory::class);
         $protocolFactory->method('createProtocol')
             ->willReturnCallback(static function (IConnection $connection) {
-                $protocol = new V5(1, $connection);
-                $protocol->serverState = ServerState::READY;
+                $serverState = new ServerState();
```
Comment (stefanak-michal) [inline tests/Unit/BoltFactoryTest.php:46]: Same as before, ServerState doesn't exist as class anymore.

File: tests/Unit/BoltFactoryTest.php (hunk 1 above)
Comment (exaby73) [reply tests/Unit/BoltFactoryTest.php:17]: Hey. Looks like there was an older version (6.x) installed via `composer.lock`. Due to this, I updated code based on that older version. I'll will update to `7.0.1` and update these parts of the code. Thanks for the great catch :)

File: tests/Unit/BoltFactoryTest.php (hunk 3 above)
Comment (exaby73) [reply tests/Unit/BoltFactoryTest.php:46]: Same issue with older version mentioned above

**Pull request merged**